server/app/src/main/resources/db/migration/V2__Updations.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE Student MODIFY COLUMN password varchar(100);
+ALTER TABLE Faculty MODIFY COLUMN password varchar(100);
```

Comment:
password is varchar(100), will it be enough to store hash generated for respective password?

Reply:
The maximum length of a bcrypt hash result is typically 60 characters. I made it to 100 to store more larger hashes in further if needed